src/worker/jobs/cron/community.ts

```diff
@@ -683,9 +683,9 @@ export async function verifyCommunityFunds(): Promise<void> {
                 const community = await CommunityService.getCommunityOnlyById(
                     communityState.communityId
                 );
-                if (community !== null) {
+                if (community !== null && community.contractAddress) {
```

Comment:
Hmm, this `verifyCommunityFunds` cron job has been turned off a while ago because it was sending too many notifications. It made sense initially. We will likely rewrite from scratch. So if you want to delete it rather than update it, please, do so.

Reply:
ok, deleted